packages/core/src/App/app.jsx

```diff
@@ -86,15 +87,17 @@ const AppWithoutTranslation = ({ root_store }) => {
             {is_translation_loaded ? (
                 <Router basename={has_base ? `/${base}` : null}>
                     <StoreProvider store={root_store}>
-                        <APIProvider>
-                            <POIProvider>
-                                <StoreProvider store={root_store}>
-                                    <ExchangeRatesProvider>
-                                        <AppContent passthrough={platform_passthrough} />
-                                    </ExchangeRatesProvider>
-                                </StoreProvider>
-                            </POIProvider>
-                        </APIProvider>
+                        <BreakpointProvider>
```

Comment:
Since library package doesnt have App.tsx to initialize the Providers, useBreakpoint doesnt work in that package. Therefore adding it higher in the tree fixes it. Plus, if other packages want to use it, they can. Gets rid of is_mobile from ui-store and isMobile from shared. 
![Screenshot 2023-12-29 at 10 36 22 AM](https://github.com/binary-com/deriv-app/assets/104053934/7ccbcac7-e720-42d9-8d10-af79baa0cf78)